<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LogQueryForm</class>
 <widget class="QWidget" name="LogQueryForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1178</width>
    <height>595</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>日志查询</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QWidget
{
     background-color: #DCDCDC;
     padding: 1px;
     border-radius: 0px;
     opacity: 0;
	 color: rgb(0, 0, 0);	
	font: 12pt ;

}

QFrame#frame
{
	border: 1px solid #183881;
	border-radius:0px;	
}


QWidget#curveChart,#heightChart,#twistChart,#rodIntervalChart1,#rodIntervalChart2,
#rodIntervalChart3,#rodIntervalChart4,#upChart,#downChart,#rodHeightChart,#gridHeightChart,#heightAndGridHeightChart
{
	border: 1px solid #183881;
}

QGroupBox
{
border: 1px solid #183881;
border-radius:0px;
margin-top:2ex;   /*文字在方框中位置的偏离度*/
font-family:Microsoft YaHei;
font:bold 18px;
color: rgb(0, 0, 0);
 
}
 
QGroupBox::title
{
subcontrol-origin:margin;
subcontrol-position:top center;
padding:0 3px;
}


QLineEdit
{
    background-color: #FFFFFF;
    selection-background-color: #D1DBCB;
    selection-color: black;
    padding: 5px;
    border-style: solid;
    border: 1px solid #76797C;
    border-radius: 5px;
    color: #000000;
}

QPushButton
{
    color: #000000;
    background-color: #BFBABA;
    border-width: 1px;
    border-color: #76797C;
    border-style: solid;
    padding: 5px;
	border-radius:5px;
}

QPushButton:hover
{
	background-color: #0550A4;
	color: rgb(255, 255, 255);
}
QPushButton:disabled
{
    background-color: #C2C46B;
    border-width: 1px;
    border-color: #76797C;
    border-style: solid;
    padding-top: 5px;
    padding-bottom: 5px;
    padding-left: 10px;
    padding-right: 10px;
    border-radius: 2px;
    color: #000000;
}
/*

QPushButton:focus {
    background-color: #D1DBCB;
    color: black;
}

QPushButton:pressed
{
    color: black;
    background-color: #D1DBCB;
    padding-top: -15px;
    padding-bottom: -17px;
}
QPushButton:checked{
    background-color: #76797C;
    border-color: #6A6969;
}
*/

QComboBox{
	border:1px solid #242424;
	border-radius:3px;
	padding:2px;
	background:none;
	color: rgb(0, 0, 0);
}

QComboBox:disabled
{
    background-color: #C2C46B;
    border-width: 1px;
    border-color: #76797C;
    border-style: solid;
    padding-top: 5px;
    padding-bottom: 5px;
    padding-left: 10px;
    padding-right: 10px;
    border-radius: 2px;
    color: #000000;
}

QComboBox::down-arrow,QDateEdit[calendarPopup=&quot;true&quot;]::down-arrow,QTimeEdit[calendarPopup=&quot;true&quot;]::down-arrow,QDateTimeEdit[calendarPopup=&quot;true&quot;]::down-arrow{
image:url(:/qss_icons/rc/add_bottom.png);
width:10px;
height:10px;
right:2px;
}

QComboBox::drop-down,QDateEdit::drop-down,QTimeEdit::drop-down,QDateTimeEdit::drop-down{
subcontrol-origin:padding;
subcontrol-position:top right;
width:15px;
border-left-width:0px;
border-left-style:solid;
border-top-right-radius:3px;
border-bottom-right-radius:3px;
border-left-color:#242424;
}

QComboBox::drop-down:on{
top:1px;
}

QTreeView::item::selected
{
	color: #ffffff;
	background-color:#0550A4;
} 
QTreeView::item::hover
{
	color: #ffffff;
	background-color:#0550A4;

}


QTableView::item:selected{
	color:#DCDCDC;
	background:qlineargradient(spread:pad,x1:0,y1:0,x2:0,y2:1,stop:0 #183881,stop:1 #183881);
}

/*
QTableView::item:hover{
color:#DCDCDC;
background:qlineargradient(spread:pad,x1:0,y1:0,x2:0,y2:1,stop:0 #183881,stop:1 #183881);
}
*/

QTableView::item{
padding:1px;
margin:0px;
}

QHeaderView::section,QTableCornerButton:section{
padding:3px;
margin:0px;
color:#DCDCDC;
border:1px solid #242424;
border-left-width:0px;
border-right-width:1px;
border-top-width:0px;
border-bottom-width:1px;
background:qlineargradient(spread:pad,x1:0,y1:0,x2:0,y2:1,stop:0 #646464,stop:1 #525252);
}

QTableView{
	border: 1px solid #183881;
	border-radius: 0px;
}

QAbstractSpinBox {
    background-color: #FFFFFF;
    color: #070707;
    selection-background-color: #1230F0;
    selection-color: black;
	border-radius:5px;
	border:1px solid #242424;
}

QAbstractSpinBox:up-button
{
    background-color: transparent;
    subcontrol-origin: border;
    subcontrol-position: center right;
}

QAbstractSpinBox:down-button
{
    background-color: transparent;
    subcontrol-origin: border;
    subcontrol-position: center left;
}

QAbstractSpinBox::up-arrow,QAbstractSpinBox::up-arrow:disabled,QAbstractSpinBox::up-arrow:off {
    image: url(:/qss_icons/rc/up_arrow_disabled.png);
    width: 10px;
    height: 10px;
}
QAbstractSpinBox::up-arrow:hover
{
    image: url(:/qss_icons/rc/up_arrow.png);
}


QAbstractSpinBox::down-arrow,QAbstractSpinBox::down-arrow:disabled,QAbstractSpinBox::down-arrow:off
{
    image: url(:/qss_icons/rc/down_arrow_disabled.png);
    width: 10px;
    height: 10px;
}
QAbstractSpinBox::down-arrow:hover
{
    image: url(:/qss_icons/rc/down_arrow.png);
}


QScrollBar:horizontal
{
    height: 15px;
    margin: 3px 15px 3px 15px;
    border: 1px transparent #2A2929;
    border-radius: 4px;
    background-color: #2A2929;
}

QScrollBar::handle:horizontal
{
    background-color: #605F5F;
    min-width: 5px;
    border-radius: 4px;
}

QScrollBar::add-line:horizontal
{
    margin: 0px 3px 0px 3px;
    border-image: url(:/qss_icons/rc/right_arrow_disabled.png);
    width: 10px;
    height: 10px;
    subcontrol-position: right;
    subcontrol-origin: margin;
}

QScrollBar::sub-line:horizontal
{
    margin: 0px 3px 0px 3px;
    border-image: url(:/qss_icons/rc/left_arrow_disabled.png);
    height: 10px;
    width: 10px;
    subcontrol-position: left;
    subcontrol-origin: margin;
}

QScrollBar::add-line:horizontal:hover,QScrollBar::add-line:horizontal:on
{
    border-image: url(:/qss_icons/rc/right_arrow.png);
    height: 10px;
    width: 10px;
    subcontrol-position: right;
    subcontrol-origin: margin;
}


QScrollBar::sub-line:horizontal:hover, QScrollBar::sub-line:horizontal:on
{
    border-image: url(:/qss_icons/rc/left_arrow.png);
    height: 10px;
    width: 10px;
    subcontrol-position: left;
    subcontrol-origin: margin;
}

QScrollBar::up-arrow:horizontal, QScrollBar::down-arrow:horizontal
{
    background: none;
}


QScrollBar::add-page:horizontal, QScrollBar::sub-page:horizontal
{
    background: none;
}

QScrollBar:vertical
{
    background-color: #2A2929;
    width: 15px;
    margin: 15px 3px 15px 3px;
    border: 1px transparent #2A2929;
    border-radius: 4px;
}

QScrollBar::handle:vertical
{
    background-color: #605F5F;
    min-height: 5px;
    border-radius: 4px;
}

QScrollBar::sub-line:vertical
{
    margin: 3px 0px 3px 0px;
    border-image: url(:/qss_icons/rc/up_arrow_disabled.png);
    height: 10px;
    width: 10px;
    subcontrol-position: top;
    subcontrol-origin: margin;
}

QScrollBar::add-line:vertical
{
    margin: 3px 0px 3px 0px;
    border-image: url(:/qss_icons/rc/down_arrow_disabled.png);
    height: 10px;
    width: 10px;
    subcontrol-position: bottom;
    subcontrol-origin: margin;
}

QScrollBar::sub-line:vertical:hover,QScrollBar::sub-line:vertical:on
{

    border-image: url(:/qss_icons/rc/up_arrow.png);
    height: 10px;
    width: 10px;
    subcontrol-position: top;
    subcontrol-origin: margin;
}


QScrollBar::add-line:vertical:hover, QScrollBar::add-line:vertical:on
{
    border-image: url(:/qss_icons/rc/down_arrow.png);
    height: 10px;
    width: 10px;
    subcontrol-position: bottom;
    subcontrol-origin: margin;
}

QScrollBar::up-arrow:vertical, QScrollBar::down-arrow:vertical
{
    background: none;
}


QScrollBar::add-page:vertical, QScrollBar::sub-page:vertical
{
    background: none;
}

QTabWidget::pane { /* The tab widget frame */
    border-top: 1px solid #C2C7CB;
    position: absolute;
    top: 0em;
}
 
QTabWidget::tab-bar {
    alignment: left;
}
 
/* Style the tab using the tab sub-control. Note that
    it reads QTabBar _not_ QTabWidget */
QTabBar::tab {
    background: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                stop: 0 #E1E1E1, stop: 0.4 #DDDDDD,
                                stop: 0.5 #D8D8D8, stop: 1.0 #D3D3D3);
    border: 1px solid #C4C4C3;
    border-bottom-color: #C2C7CB; /* same as the pane color */
    border-top-left-radius: 10px;
    border-top-right-radius: 10px;
    min-width: 40ex;
    padding: 2px;
}
 
QTabBar::tab:selected, QTabBar::tab:hover {
    background-color: #0550A4;
	color: rgb(255, 255, 255);
}
 
QTabBar::tab:selected {
	background-color:#00aa00;
	color: rgb(255, 255, 255);
}

</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout" stretch="0,0">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="TitleWidget" name="titleWidget" native="true"/>
   </item>
   <item>
    <widget class="QFrame" name="frame">
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QGridLayout" name="gridLayout_3">
      <item row="0" column="0" colspan="2">
       <widget class="QGroupBox" name="groupBox">
        <property name="title">
         <string/>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <widget class="QLabel" name="label_4">
           <property name="minimumSize">
            <size>
             <width>50</width>
             <height>40</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>50</width>
             <height>40</height>
            </size>
           </property>
           <property name="text">
            <string>时间</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QDateEdit" name="startTime">
           <property name="minimumSize">
            <size>
             <width>150</width>
             <height>40</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>16777215</width>
             <height>40</height>
            </size>
           </property>
           <property name="displayFormat">
            <string>yyyy-MM-dd </string>
           </property>
           <property name="calendarPopup">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_5">
           <property name="minimumSize">
            <size>
             <width>15</width>
             <height>0</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>15</width>
             <height>16777215</height>
            </size>
           </property>
           <property name="text">
            <string>-</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QDateEdit" name="endTime">
           <property name="minimumSize">
            <size>
             <width>150</width>
             <height>40</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>16777215</width>
             <height>40</height>
            </size>
           </property>
           <property name="displayFormat">
            <string>yyyy-MM-dd </string>
           </property>
           <property name="calendarPopup">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_2">
           <property name="minimumSize">
            <size>
             <width>50</width>
             <height>40</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>50</width>
             <height>40</height>
            </size>
           </property>
           <property name="text">
            <string>型号</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="TypeList">
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>40</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>16777215</width>
             <height>40</height>
            </size>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="queryBtn">
           <property name="minimumSize">
            <size>
             <width>150</width>
             <height>40</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>350</width>
             <height>150</height>
            </size>
           </property>
           <property name="text">
            <string>查询</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="export_data_btn">
           <property name="minimumSize">
            <size>
             <width>150</width>
             <height>40</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>350</width>
             <height>150</height>
            </size>
           </property>
           <property name="text">
            <string>导出</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QGroupBox" name="groupBox_3">
        <property name="title">
         <string>2#传感器数据</string>
        </property>
        <layout class="QGridLayout" name="gridLayout_2">
         <item row="0" column="0">
          <widget class="QTableWidget" name="result_tableWidget_2">
           <attribute name="horizontalHeaderDefaultSectionSize">
            <number>300</number>
           </attribute>
           <column>
            <property name="text">
             <string>采样点数</string>
            </property>
           </column>
           <column>
            <property name="text">
             <string>角度(°)</string>
            </property>
           </column>
           <column>
            <property name="text">
             <string>测量数据(mm)</string>
            </property>
           </column>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QGroupBox" name="groupBox_2">
        <property name="title">
         <string>1#传感器数据</string>
        </property>
        <layout class="QGridLayout" name="gridLayout">
         <item row="0" column="0">
          <widget class="QTableWidget" name="result_tableWidget_1">
           <attribute name="horizontalHeaderDefaultSectionSize">
            <number>300</number>
           </attribute>
           <column>
            <property name="text">
             <string>采样点数</string>
            </property>
           </column>
           <column>
            <property name="text">
             <string>角度(°)</string>
            </property>
           </column>
           <column>
            <property name="text">
             <string>测量数据(mm)</string>
            </property>
           </column>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>TitleWidget</class>
   <extends>QWidget</extends>
   <header>titlewidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
